src/tomato/drivers/biologic/main.py

```diff
@@ -21,7 +97,7 @@ def get_status(
     dllpath: str = None,
     lockpath: str = None,
```

Comment:
I think if you add `timeout` (if necessary) and `retries` as here (along with type and default values), they should get fished out of the settings dictionary, which also contains the `dllpath` and `lockpath` (see the `settings.toml` file in your installation.